components/camera/align/extrinsics.go

```diff
@@ -48,7 +48,7 @@ func init() {
 				if err != nil {
 					return nil, err
 				}
-				return camera.FromVideoSource(conf.ResourceName(), src), nil
+				return camera.FromVideoSource(conf.ResourceName(), src, logger), nil
```

Comment:
This should be good! `FromVideoSource` returns a `sourceBasedCamera` which already has `resource.Named` so we should be covered here by adding in `logger` here.

Reply:
Great! Just to make sure, this is not a breaking change right? We did change some method things like adding that logger param for `FromVideoSource`. I updated it all in RDK, but would it cause issues for modules and other repos?

Reply:
@bhaney just want to confirm ^

Reply:
This may break some module's sources potentially if they were written in Go

Reply:
Or rather, it will break modules that build cameras. Maybe check with every published camera module written in Go and also create a PR in those repos